### DISCUSSION TITLE
Maximum requests Elsa Workflow v2 can execute

### DISCUSSION THREAD

**Discussion by Rathna314** [elsa-workflows/elsa-guides#46] [Q&A]
Is there a maximum limit on how many request Elsa workflow 2.14 can process and how do we set that limit. 
I am using a custom activity and using send http request to another API for backend process for each request. I can run the workflow for requests less than 1000 but it runs into problems for a larger set. Please help.

**Comment by mosiera98** [comment]
It's strange to me why no one answers these questions with so many participants.
